src/routes/Send.tsx

```diff
@@ -451,11 +451,15 @@ export function Send() {
     function processInvoice(source: ParsedParams & { invoice: string }) {
         state.mutiny_wallet
             ?.decode_invoice(source.invoice!)
-            .then((invoice) => {
+            .then(async (invoice) => {
                 if (invoice.expire <= Date.now() / 1000) {
                     navigate("/search");
                     throw new Error(i18n.t("send.error_expired"));
                 }
+                const isHodl = await MutinyWallet.is_potential_hodl_invoice(
+                    source.invoice!
```

Comment:
why is source here and not not `invoice`?

Reply:
invoice isn't a string, it is a `MutinyInvoice`